### ISSUE TITLE
Add option to control ANSI color codes

### ISSUE THREAD

**Issue by shadycuz** [devcontainers/cli#253]
I cant figure out how to disable terminal colors? I get this in Jenkins:

```
[1m[31m#[1m[38;2;86;156;214m13[39m[22m naming to docker.io/library/vsc-devcontainer-example-c17e9ad94bc542f72c1906e018d524aa 0.0s done[39m[22m
[1m[31m#[1m[38;2;86;156;214m13[39m[22m DONE 0.1s[39m[22m
[1m[31m[39m[22m
[1m[31m#[1m[38;2;86;156;214m14[39m[22m exporting cache[39m[22m
[1m[31m#[1m[38;2;86;156;214m14[39m[22m preparing build cache for export done[39m[22m
[1m[31m#[1m[38;2;86;156;214m14[39m[22m DONE 0.0s[39m[22m
[1m[31m[39m[22m
[[1m[38;2;99;143;79m414703 ms[39m[22m] Start: Run: docker run --sig-proxy=false -a STDOUT -a STDERR --mount 
```

**Comment by shadycuz** [comment]
@chrmarti Was this released in version `0.23.2`? I'm still having the issue. It would be nice to have a flag or environment variable to control color output.

**Comment by chrmarti** [comment]
I only now released this with 0.24.0. Let me know if that fixes it for you. Thanks.
